src/components/Messages/Chatroom.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react'
+import {connect} from 'react-redux'
+import {Image} from 'semantic-ui-react'
+import {Message} from './Message.js'
+import { makeMessageRead } from '../../actions/useractions.js'
+
+class Chatroom extends React.Component {
+  componentDidMount(){
+    const chatroom = this.props.chatrooms.find(cr => cr.id === this.props.chatroomId)
+    const lastMessage = chatroom?.messages[chatroom.messages.length - 1]
+    if (lastMessage?.read === false && lastMessage.user_id !== this.props.currentUser.id){
+      console.log("making message read")
+      this.props.makeMessageRead(lastMessage.id)
+    }
+  }
+  render(){
+    const chatroom = this.props.chatrooms.find(cr => cr.id === this.props.chatroomId)
+    const otherUser = chatroom.users.find(u => u.id !== this.props.currentUser.id)
+    return (
+      <div style={{height: "80vh"}}>
```

Comment:
I would say it's a good idea to separate css from js, so i'd replace that style with something in the scss/style folder. Maybe add a chatroom.scss in style/components, import it into index.scss, and then make any style changes there